### ISSUE TITLE
Running a single test fails

### ISSUE THREAD

**Issue by Invertisment** [clojure-emacs/cider#2296]
When I place my cursor on a test and I execute `Leader-t-t` or `spacemacs/cider-test-run-focused-test` I should get a test run in REPL.
It worked about two weeks ago (Now it's 21th of May) and it doesn't go away.

Is it an issue with my setup or a bug?

I get this error:

```
Exception in thread "nREPL-worker-3" 
java.lang.IllegalArgumentException: Symbol must be namespace-qualified
	at clojure.lang.Var.find(Var.java:140)
	at clojure.core$find_var.invokeStatic(core.clj:2012)
	at clojure.core$find_var.invoke(core.clj:2007)
	at clojure.core$comp$fn__5529.invoke(core.clj:2561)
	at clojure.core$keep$fn__8200.invoke(core.clj:7257)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.RT.seq(RT.java:528)
	at clojure.core$seq__5124.invokeStatic(core.clj:137)
	at clojure.core$seq__5124.invoke(core.clj:137)
	at cider.nrepl.middleware.util.coerce$var_query$fn__17281.invoke(coerce.clj:37)
	at clojure.core$update.invokeStatic(core.clj:6116)
	at clojure.core$update.invoke(core.clj:6108)
	at clojure.lang.AFn.applyToHelper(AFn.java:160)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:661)
	at clojure.core$apply.invoke(core.clj:652)
	at cider.nrepl.middleware.util.coerce$update_some.invokeStatic(coerce.clj:7)
	at cider.nrepl.middleware.util.coerce$update_some.doInvoke(coerce.clj:4)
	at clojure.lang.RestFn.invoke(RestFn.java:442)
	at cider.nrepl.middleware.util.coerce$var_query.invokeStatic(coerce.clj:37)
	at cider.nrepl.middleware.util.coerce$var_query.invoke(coerce.clj:33)
	at cider.nrepl.middleware.test$handle_test_var_query_op$fn__17450$fn__17451$fn__17452.invoke(test.clj:284)
	at cider.nrepl.middleware.test$handle_test_var_query_op$fn__17450$fn__17451.invoke(test.clj:279)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:657)
	at clojure.core$with_bindings_STAR_.invokeStatic(core.clj:1965)
	at clojure.core$with_bindings_STAR_.doInvoke(core.clj:1965)
	at clojure.lang.RestFn.invoke(RestFn.java:425)
	at cider.nrepl.middleware.test$handle_test_var_query_op$fn__17450.invoke(test.clj:277)
	at clojure.tools.nrepl.middleware.interruptible_eval$run_next$fn__4551.invoke(interruptible_eval.clj:190)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

![screenshot at 2018-05-21 17 53 18](https://user-images.githubusercontent.com/1641263/40314188-6f8bce1a-5d20-11e8-8292-8d39456c8d2b.png)

Versions:
```
CIDER 0.17.0snapshot (package: 20180506.1409)
GNU Emacs 25.3.2 (x86_64-pc-linux-gnu, GTK+ Version 3.18.9) of 2017-09-12
Spacemacs
Leiningen 2.8.1 on Java 1.8.0_171 Java HotSpot(TM) 64-Bit Server VM
Oracle Java 8 and Java 9 performs same thing
Ubuntu 16.04.4 LTS
```

**Comment by bbatsov** [comment]
Can you upgrade to the latest version? This was bug we introduced and (supposedly) fixed shortly before the release of 0.17 (Andalucia).

**Comment by Invertisment** [comment]
Thanks. Works with `CIDER 0.18.0snapshot (package: 20180521.1210)`
